### PR TITLE
Add Vue transformer

### LINKED ISSUE #6: Vue Transformer
See #7

### PR DESCRIPTION
**Pull request by m1guelpf**
Closes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +58,11 @@ The following transformers are availble:
 - `addTailwindCss` - adds link to tailwindui.css
 - `prefixSrc` - adds `https://tailwindui.com` to any img src attribute that needs it
 - `useInter` - adds link to Inter font css and styles
+- `convertVue` - converts HTML component into a Vue component
+  - Use `.env` key `VUE_OUTPUT` to specify which folder to save the React files to
```
Comment (taftse) [inline README.md:62]: Should this be Vue Files ?

Comment (m1guelpf) [reply README.md:62]: They are called Vue components as far as I know but can rename if that terminology is preferred

Comment (m1guelpf) [reply README.md:62]:
```suggestion
  - Use `.env` key `VUE_OUTPUT` to specify which folder to save the Vue files to
```

**Comment by vesper8** [comment]
this is so awesome!

**Pull request merged**